api/src/controllers/upgrade.js

```diff
@@ -1,11 +1,14 @@
 const auth = require('../auth');
 const serverUtils = require('../server-utils');
 
-const service = require('../services/upgrade');
+const service = require('../services/horti/upgrade');
```

Comment:
Can we drop the "horti" name altogether? Nobody gets the joke  :/

Reply:
Yea haha definitely.